src/transition_tracer/transition_tracer.cpp

```diff
@@ -138,10 +138,14 @@ TransitionTracer::TransitionTracer(user_input &input)
             new_transition_data.crit_false_vev =
                 pair.false_phase.Get(pair.crit_temp).point;
 
+            (void)CheckMassRatio(
+                input, new_transition_data.crit_false_vev, pair.crit_temp);
```

Comment:
Why do you need these function calls?

Reply:
The `CheckMassRatio` function prints the values of `m^2/T^2`, coloured by how large the ratio is, and returns the maximal ratio. The latter we use in `Test-gw.cpp` to test this function, the former is to give at least some information about the validity of the high-T expansion.

Reply:
We call it for each characteristic temperature that we calculate.